### PR TITLE
Feature/update mainnet tests

### PR DESCRIPTION
**Pull request by lcamargof**
## Description of changes
Use already deployed RSR and ForkSpell in the tests

### CODE REVIEW

File: test/integration/RSR-integration.test.ts [hunk 1]
```diff
@@ -22,15 +22,18 @@ const mockSiphons = UPGRADE_SIPHONS.slice(0, 5).map((siphon, index) => ({
 
 // Relevant addresses (Mainnet)
 const RSR_PREVIOUS_ADDRESS = '0x8762db106b2c2a0bccb3a80d1ed41273552616e8'
+const RSR_ADDRESS = '0x320623b8E4fF03373931769A31Fc52A4E78B5d70'
+const FORK_SPELL_ADDRESS = '0x53b968CAF6A1715cAA8FA3f0fb63E8D09a6433e1'
+const DEPLOYER_ADDRESS = '0x9f59852571E708791276Acf110D0B31be2756947'
```
Comment (julianmrodri) [inline test/integration/RSR-integration.test.ts:27]: Im not sure this is relevant right?

**Review by julianmrodri — APPROVED** [review]
lgtm

File: test/integration/RSR-integration.test.ts (hunk 1 above)
Comment (tbrent) [reply test/integration/RSR-integration.test.ts:27]: Shouldn't need Deployer address.

Comment (tbrent) [reply test/integration/RSR-integration.test.ts:27]: Mmmm I see how it's being used...seems fine.

Comment (lcamargof) [reply test/integration/RSR-integration.test.ts:27]: Yeah, considering that we are no deploying it in the test and that checking the owner is part of the test, checking if the owner is the companySafe doesn't really mean anything.

**Pull request closed without merging**